src/_pytest/fixtures.py

```diff
@@ -1489,8 +1488,8 @@ def getfixtureinfo(
         else:
             argnames = ()
 
-        usefixtures = itertools.chain.from_iterable(
-            mark.args for mark in node.iter_markers(name="usefixtures")
+        usefixtures = tuple(
+            arg for mark in node.iter_markers(name="usefixtures") for arg in mark.args
         )
         initialnames = tuple(usefixtures) + argnames
         fm = node.session._fixturemanager
```

Comment:
```suggestion
        return FuncFixtureInfo(
            argnames,
            *node.session._fixturemanager.getfixtureclosure(
                tuple(
                    arg
                    for mark in node.iter_markers(name="usefixtures")
                    for arg in mark.args
                )
                + argnames,
                node,
                ignore_args=self._get_direct_parametrize_args(node),
            )
        )
```

Reply:
:-1: from me, the original code is much more readable in my opinion.

Reply:
if that's too much of a mega-expression, I still think it's worth doing:


```python
initialnames = (
    tuple(arg for mark in node.iter_markers(name="usefixtures") for arg in mark.args)
    + argnames
)
```

Reply:
I think it's nice to build it in two steps, so I'll keep it as is.

I did forget to remove the `tuple(..)` from the existing line though, will remove it.